packages/data/src/registry.js

```diff
@@ -270,15 +280,17 @@ export function createRegistry( storeConfigs = {}, parent = null ) {
 	 * @param {StoreDescriptor} store Store descriptor.
 	 */
 	function register( store ) {
```

Comment:
I possibly mis-understand this! The extra instance from the second bundle wouldn't ever be instantiated. If that's a problem, should we return the already registered store from `registerStoreInstance` in each of these functions? Then, at least the second store instance could get assigned to the one stored in the registry.

I guess that would make the consumer look more like `export const store = register( createReduxStore( ... ) )`. 

Happy to be corrected on this; I don't understand the internals of store registration that well :)